picby_graphql_api/src/index.ts

```diff
@@ -15,12 +20,38 @@ import { RegisterResolver } from './modules/user/Register';
 
   const apolloServer = new ApolloServer({
     schema: await buildSchema({
-      resolvers: [RegisterResolver],
-      validate: true
+      resolvers: [RegisterResolver, LoginResolver, MeResolver],
+      validate: true,
+
     }),
     context: ({ req, res }) => ({ req, res })
   });
 
+  const RedisStore = connectRedis(session);
+
+  app.use(cors({
+    credentials: true,
+    origin: '*'
+  }))
+  
+  app.use(
+    session({
+      store: new RedisStore({
+        client: redis as any
+      }),
+      name: 'qid',
+      secret: 'asdasda',
```

Comment:
we probably should set variables at the top of the file, test ones too
+ more a tip than a comment but i think we should take care of the import order, at the top libs then our files